GrandCentralBoard/Core/NetworkingBridge.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Created by Oktawian Chojnacki on 11.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Alamofire
+
+
+public enum Method: String {
+    case OPTIONS, GET, HEAD, POST, PUT, PATCH, DELETE, TRACE, CONNECT
+}
+
+public protocol JSONRequesting {
+    func requestJSON(method: Method, url: NSURL, parameters: [String : AnyObject]?, headers: [String : String]?, completion: (Result<AnyObject>) -> Void)
+}
+
+public protocol DataRequesting {
+    func requestData(method: Method, url: NSURL, parameters: [String : AnyObject]?, headers: [String : String]?, completion: (Result<NSData>) -> Void)
+}
+
+public final class NetworkingBridge {
+    private let manager: Alamofire.Manager
+
+    public init(manager: Alamofire.Manager = Alamofire.Manager.sharedInstance) {
+        self.manager = manager
+    }
+}
+
+extension NetworkingBridge : JSONRequesting {
+
+    public func requestJSON(method: Method, url: NSURL, parameters: [String : AnyObject]?, headers: [String : String]?, completion: (Result<AnyObject>) -> Void) {
+        let method = Alamofire.Method(fromMethod: method)
+        manager.request(method, url, parameters: parameters, headers: headers).responseJSON { response in
```

Comment:
I believe we also need to set `encoding: .JSON` (had a problem with this today). We are requesting JSON, so parameters should be encoded the same way.